docs/srt-live-transmit.md

```diff
@@ -208,36 +215,72 @@ explicitly. Note also special cases of the **host** and **port** parts
-- **pbkeylen**:  Crypto key len in bytes {16,24,32} Default: 16 (128-bit)
-- **tlpktdrop**: Whether to drop packets that are not delivered on time. Default is on.
-- **conntimeo**: Connection timeout (in ms). Caller default: 3000, rendezvous (x 10)
-
-For the complete list of options, please refer to the SRT header file `srt.h` and search for `SRT_SOCKOPT` enum type. Please note that the set of available options may be version dependent. All options are available under the lowercase name of the option without the `SRTO_` prefix. For example, `SRTO_PASSPHRASE` can be set using
-a **passphrase** parameter. The mapping table `srt_options` can be found in `common/socketoptions.hpp` file.
+All other parameters are SRT socket options.
+
+| URI param   | SRT Option | Description | Possible values |
+| ----------- | ---------- | ----------- | --------------- |
+| `transtype` | `SRTO_TRANSTYPE` | Transmission type | {`live`, `file`} |
+| `maxbw`     | `SRTO_MAXBW` | Bandwidth limit in bytes | [0, ..] |
+| `pbkeylen`  | `SRTO_PBKEYLEN` | Crypto key length in bytes. | {16, 24, 32} |
+| `passphrase`| `SRTO_PASSPHRASE` | Password for the encrypted transmission. | string |
+| `mss` | `SRTO_MSS` | MTU size | integer |
+| `fc` | `SRTO_FC` | Flow control window size | integer |
+| `sndbuf` | `SRTO_SNDBUF` | Sender buffer size in bytes | integer |
+| `rcvbuf` | `SRTO_RCVBUF` | Receiver buffer size in bytes | integer |
+| `linger` | `SRTO_LINGER` | Link linger value | integer |
+| `ipttl` | `SRTO_IPTTL` | Defines IP socket "time to live" option | bool |
+| `iptos` | `SRTO_IPTOS` | Defines IP socket type of service | bool |
+| `inputbw` | `SRTO_INPUTBW` | Defines input bandwidth in bytes | integer | 
+| `oheadbw` | `SRTO_OHEADBW` | limits bandwidth overhead, percents | integer |
+| `latency` | `SRTO_LATENCY` | Defines the maximum accepted transmission latency. Should be >= 2.5 times the RTT (default: 120ms; when both parties set different values, the maximum of the two is used for both). | integer |
+| `tsbpdmode` |  `SRTO_TSBPDMODE` | Enables/disables timestamp-based packet delivery mode | bool |
+| `tlpktdrop` | `SRTO_TLPKTDROP` | Enables/disables the Too-Late Packet Drop logic | bool |
+| `snddropdelay` | `SRTO_SNDDROPDELAY` | Sender's delay before dropping packets in ms | integer |
+| `nakreport` | `SRTO_NAKREPORT` | Enables/disables periodic NAK reports | {{`true` , `on` `1`}, {`false`, `off`, `0`}} |
+| `conntimeo` | `SRTO_CONNTIMEO` | Connection timeout (in ms). Caller default: 3000, rendezvous (x 10) | |
+| `lossmaxttl` | `SRTO_LOSSMAXTTL` | Packet reorder tolerance. | integer |
+| `rcvlatency` | `SRTO_RCVLATENCY` | Receiver-side latency (in ms). | integer |
+| `peerlatency` | `SRTO_PEERLATENCY` | Minimum receiver latency to be requested by sender (in ms). | integer |
+| `minversion` | `SRTO_MINVERSION` | Minimum SRT library version of a peer. | maj.min.rev |
+| `streamid` | `SRTO_STREAMID` | Stream ID. | string |
```

Comment:
Stream ID (settable in caller mode only, visible on the listener peer).

Not sure if necessary, but at least avoids confusion and Question-type issues.